test/built-ins/Temporal/Duration/prototype/round/february-leap-year.js

```diff
@@ -18,21 +18,21 @@ const options = {
   relativeTo,
 };
 
-const twoDaysLessThanFourYears = new Temporal.Duration(3, 11, 0, 27);
+const twoDaysLessThanFourYears = new Temporal.Duration(3, 11, 0, 29);
```

Comment:
Is this still two days less than four years, then? Does it balance up all of a sudden if it's actually two days less than four years? I wouldn't expect that change.

Reply:
Resolved, see top level comment!